src/api/issued_book/endpoints.py

```diff
@@ -0,0 +1,111 @@
+from http import HTTPStatus
+from typing import Dict, Tuple
+
+from flask import request
+from flask_restx import Resource
+
+from api.issued_book import schemas
+from helpers.responses import failure_response, success_response
+from model.book import Book
+from model.issued_book import IssuedBook
+from model.user import User
+
+from . import api
+
+
+@api.route("/")
+class IssueBook(Resource):
+    @api.doc("Issue a book")
+    @api.expect(schemas.issued_book_expect, validate=True)
+    @api.marshal_with(schemas.issued_book_response, skip_none=True)
+    def post(self) -> Tuple[Dict, int]:
+        """
+        Issue a book to a user
+
+        Returns:
+            Issued Book
+        """
+        api.logger.info("Issue book")
+
+        book = Book.get_by_id(api.payload["book_id"])
+        if not book:
+            return failure_response(["Book not found."], HTTPStatus.NOT_FOUND)
+
+        if book.available_quantity <= 0:
+            return failure_response(["Book not available."], HTTPStatus.BAD_REQUEST)
```

Comment:
`err = Book not available`
`return failure_response(err, HTTPStatus.BAD_REQUEST)`